brave/brave-ios
# Fix #2129: Update images for PIN modal

## Summary of Changes


This pull request fixes #2129 

## Submitter Checklist:


## Test Plan:
1. Navigate to https://demo.yubico.com/u2f
2. Use a yubikey with PIN
3. Verify the updated image is displayed in the modal

## Screenshots:

**Current**:

![Screen Shot 2021-01-12 at 3 32 21 PM](https://user-images.githubusercontent.com/1903815/104387412-e458ba80-54eb-11eb-94b5-9d8522e7f6f8.png)

**New:**

![Screen Shot 2021-01-12 at 3 32 02 PM](https://user-images.githubusercontent.com/1903815/104387421-e9b60500-54eb-11eb-936a-5a7b7a4c142a.png)

## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `release-notes/(include|exclude)`
  - `bug` / `enhancement`
